### PR TITLE
Update to Newthrone: switching from WETH to THRO 

### PR DESCRIPTION
**Pull request by ItsMeTazz**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

File: projects/newthrone/index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-const ADDRESSES= require("../helper/coreAssets.json");
 
 const GAME_CONTRACT = '0x61C3A357bc3ca51b80eCD36CB1Ae37e5465C6701'
 
 module.exports = {
-  methodology: 'WETH tokens in the Game contract, which represents the current balance of the game (total spent - total claimed).',
+  methodology: 'THRO tokens in the Game contract, which represents the current balance of the game (total spent - total claimed).',
   base: {
-    tvl: sumTokensExport({ owner: GAME_CONTRACT, tokens: [ADDRESSES.base.WETH]}),
+    tvl: sumTokensExport({ owner: GAME_CONTRACT, tokens: ['0x0f929C29dcE303F96b1d4104505F2e60eE795caC']})
```
Comment (g1nt0ki) [inline projects/newthrone/index.js:8]: hi @ItsMeTazz we cant count THRO as tvl, can instead set tvl to 0 and count this under staking?

Comment (ItsMeTazz) [reply projects/newthrone/index.js:8]: Hey @g1nt0ki What would be the result of doing this? 

The TLDR is that players invest THRO, which is redistributed over time from the game contract. Our TVL represents the total of THRO in that game contract.

Comment (g1nt0ki) [reply projects/newthrone/index.js:8]: yes, I understand, but we can count this investment as staking, not as tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Comment (ItsMeTazz) [reply projects/newthrone/index.js:8]: Okay, I don't know what the visual/numeral result of doing this is, but as long as we can track the total of THRO in the contract, I think we're good.

**Comment by llamatester** [comment]
The adapter at projects/newthrone exports TVL: 
        
 
 
    base                      0
    base-staking              0
    staking                   0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/newthrone exports TVL: 
        
 
 
    base                      0
    base-staking              165.53 k
    staking                   165.53 k
    
    total                    0

File: projects/newthrone/index.js (hunk 1 above)
Comment (ItsMeTazz) [reply projects/newthrone/index.js]: @g1nt0ki How long does it usually take to be updated on the live website? Still seeing 0, wondering if it updates every X hours, or if it should be instantaneous. 

Thanks again for your time!

Comment (g1nt0ki) [reply projects/newthrone/index.js]: did you toggle staking? 
image

**Pull request merged**